README.md

```diff
@@ -111,13 +139,13 @@ curl -X 'POST' \
   | jq -C . | less -R
 ```
 
-#### Languages
+### Languages
 
 You can also specify what languages to use for OCR with the `languages` kwarg. See the [Tesseract documentation](https://github.com/tesseract-ocr/tessdata) for a full list of languages and install instructions. OCR is only applied if the text is not already available in the PDF document.
 
 ```
 curl -X 'POST' \
-  'https://api.unstructured.io/general/v0/general' \
+  'https://api.unstructuredapp.io/general/v0/general' \
```

Comment:
Consider using PDF here? If so, also consider adding the additional -F parameters for PDFs, as listed above.